EWL-4673: Add Topics Page VRT test

designerneil opened the pull request:
## Ticket(s)

**Jira Ticket**
- [EWL-4673: SG2 | Write VRT tests for the Topics pages elements](https://issues.ama-assn.org/browse/EWL-4673)

## Description
Add VRT for the topics page so we can see any visual regressions that occur

## To Test
- [ ] `gulp serve`
- [ ] In another terminal window run `gulp backstop --filter=topics-page` 
- [ ] Observe all VRT test pass
- [ ] Make a code change to the topics page in your local environment
- [ ] Run `gulp backstop`
- [ ] Observe backstop errors

## Visual Regressions

13 new backstop tests were added:
- page title
- article stub
- article stub as inline
- article stub as related 
- article stub as small image
- article stub as title only
- article stub as video
- lead news
- membership
- membership as indented
- membership with subtitle
- tools block
- footer

87 new backstop ref images were created to test all elements on the topics page for visual regression

To further prevent regression on the topics page the following classes will be observed by backstop:
`".ama__page-title", ".ama__membership ", ".ama__tools-block", ".ama__promo", ".ama__article-stub", ".ama__lead-news", ".ama__footer"`

## Relevant Screenshots/GIFs
N/A

## Remaining Tasks
N/A

## Additional Notes
You will get backstop errors because the date molecule never stays the same due to the fact it uses the current date and not a static one.

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

designerneil committed:
EWL-4673: Adds Topics Page VRT test

labbydev commented on styleguide/backstop.json:
```diff
@@ -232,7 +232,7 @@
       "hideSelectors": [],
       "removeSelectors": [],
-      "selectors": [],
+      "selectors": [".ama__page-title", ".ama__membership ", ".ama__tools-block", ".ama__promo", ".ama__article-stub", ".ama__lead-news", ".ama__footer"],
```
@designerneil when I did this approach during the travis testing the FEDs decided that we should create a scenario for each pattern to test it on it's own instead of just on the page since patterns may be used on multiple pages and while some patterns may be used in Drupal they may not be on the example page. Please separate these out into individual scenarios.

designerneil committed:
EWL-4673: Adds VRT tests:
- page title
- article stub
-  article stub as inline
- article stub as related
- article stub as small image
- article stub as title only
- article stub as video
- lead news
- membership
- membership as indented
- membership with subtitle
- tools block
- footer

designerneil committed:
EWL-4673: Adds VRT tests:
- page title
- article stub
-  article stub as inline
- article stub as related
- article stub as small image
- article stub as title only
- article stub as video
- lead news
- membership
- membership as indented
- membership with subtitle
- tools block
- footer

designerneil committed:
Merge branch 'develop' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4673-topics-vrt-test

# Conflicts:
#	styleguide/backstop_data/bitmaps_reference/backstop_default_news_0_document_3_bp-large.png

designerneil committed:
Merge branch 'develop' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4673-topics-vrt-test

# Conflicts:
#	styleguide/backstop_data/bitmaps_reference/backstop_default_organisms-masthead-with-subtitle_0_ama__masthead_1_bp-small.png
